DorisProject/src/app/pages/add-category/add-category.component.ts

```diff
@@ -32,22 +41,30 @@ export class AddCategoryComponent implements OnInit {
   save() {
     this.mostrar = false;
     this.mostrarError = false;
-    const category = { "name": this.addCategoryForm.get('name').value, "description": this.addCategoryForm.get('description').value, "active": "Y" };
+    const category = {
+      'name': this.addCategoryForm.get('name').value,
```

Comment:
Codacy has a fix for the issue: [Replace `'name'` with `name`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
      name: this.addCategoryForm.get('name').value,
```